client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.test.js

```diff
@@ -46,9 +46,76 @@ describe('DataFilesToolbar', () => {
     expect(getByText(/Trash/)).toBeDefined();
     expect(queryByText(/Make Public/)).toBeFalsy();
   });
-});
 
-describe('DataFilesToolbar', () => {
+  it('does not render unnecessary buttons in Community Data', () => {
+    const {getByText, queryByText} = renderComponent(
+      <DataFilesToolbar scheme="community" api="tapis" />,
+      mockStore({
+        workbench: { config: {
+          extract: '',
+          compress: ''
+        } },
+        files: { selected: { FilesListing: [] } },
+        listing: { selected: { FilesListing: [] } },
+        systems: systemsFixture
+      }),
+      createMemoryHistory()
+    );
+
+    expect(queryByText(/Rename/)).toBeFalsy();
+    expect(queryByText(/Move/)).toBeFalsy();
+    expect(getByText(/Copy/)).toBeDefined();
+    expect(getByText(/Download/)).toBeDefined();
+    expect(queryByText(/Trash/)).toBeFalsy();
+    expect(queryByText(/Make Public/)).toBeFalsy();
+  });
+
+  it('do not render unnecessary buttons in Public Data', () => {
```

Comment:
Could you change this and [line 96](https://github.com/TACC/Core-Portal/pull/436/files#diff-09532d63ab073efaff673660854440817a09702e7903a9c21ee7752646ea7830R96) as well from `do` to `does`?
```suggestion
  it('does not render unnecessary buttons in Public Data', () => {
```